plugin.json

```diff
@@ -1,7 +0,0 @@
-{
-    "name": "ISIC Archive",
-    "description": "The ISIC dermoscopy image archive.",
-    "url": "https://github.com/ImageMarkup/isic-archive",
-    "version": "1.0.0",
-    "dependencies": ["gravatar", "large_image", "oauth", "worker"]
```

Comment:
We may want to drop `gravatar` in the future, but let's not make this PR change it. So, we should add `girder_gravatar` to this plugin's dependencies.